custom_reports/columns.rst

```diff
@@ -47,3 +47,62 @@ dimensions to metrics or different dimensions.
 
 .. include:: columns__dimension_to_metric_transformations.rst
 .. include:: columns__dimension_to_dimension_transformations.rst
+
+Exporter
+--------
+
+The tables in the following sections shows all columns that may be easily used in Exporter config.
+For sessions and events scope there is preferred list of columns due to performance and reliability reasons.
+
+.. note::
+    The table below presents a short description of the selected fields.
+
++--------------------------------+-------------------------------------------------------+
+| Column                         | Description                                           |
++================================+=======================================================+
+| website_name                   | Binary format (16 bytes).                             |
```

Comment:
I'm afraid we misunderstood - these descriptions aren't for exporter.
in exporter `website_name` and locations aren't in binary format, timestamp is not in UTC.. tbh I'd remove this whole section as it doesn't make too much sense to me

Reply:
removed